### PR TITLE
Remove 'User Media in an IFrame' section and use new 'user media enabled flag'

### PR DESCRIPTION
**Pull request by adam-be**
Proposed fix for issue #353

### CODE REVIEW

**Comment by stefhak** [comment]
LGTM

**Comment by alvestrand** [comment]
LGTM, tagging @domenic and @dontcallmedom for verification.

File: getusermedia.html [hunk 1]
```diff
@@ -2494,6 +2494,8 @@
     Agents to remove these APIs entirely when on an insecure origin, as long as
     they remove all of them at once (e.g., they should not leave just the
     prefixed version available on insecure origins).</p>
+    <p>All Documents have an <dfn>user media enabled flag</dfn>. This
```
Comment (dontcallmedom) [inline getusermedia.html]: s/an/a/

**Comment by domenic** [comment]
LGTM. Will this be published to https://w3c.github.io/mediacapture-main/ soon?

File: getusermedia.html (hunk 1 above)
Comment (adam-be) [inline getusermedia.html]: I'll push a fix for this one.

**Pull request merged**